app/utils/tx-utils.ts

```diff
@@ -9,3 +9,7 @@ export function getRecipientAddress(tx: Transaction) {
   if (tx.tx_type !== 'token_transfer') return null;
   return tx.token_transfer.recipient_address;
 }
+
+export function isLockTx(tx: Transaction): boolean {
+  return tx.tx_type === 'contract_call' && tx.contract_call.contract_id.includes('pox');
```

Comment:
Can other people submit contracts with pox in?

Reply:
Good catch, yes they can. I've refactored to match the exact contract ID of the pox contract.